cg/src/domains/processors/processors.service.js

```diff
@@ -0,0 +1,115 @@
+const { db } = require('../../db');
```

Comment:
Rename to `SubjectsService`, but keep in the same domain.

Reply:
I think there should be some discussion in place, cause this isn't DRY if we need to maintain two very similar classes in two separate domains. Let's think on how the processors' domain should interact with the subject's data. Maybe we can simplify things.

Reply:
Ok